README.md

```diff
@@ -332,3 +332,17 @@ llm-analysis <func-name> <args> <kwargs>
 
 If you have a previous version of the package installed, you may need to
 uninstall and reinstall the package to get the new console command.
+
+# Cite us
+```
+@misc{thakur2024judgingjudgesevaluatingalignment,
```

Comment:
`thakur2024judgingjudgesevaluatingalignment` -> `thakur2024judging`

Suggested name is too long